src/main/java/net/earthcomputer/clientcommands/command/SignSearchCommand.java

```diff
@@ -30,10 +29,10 @@ public static void register(CommandDispatcher<FabricClientCommandSource> dispatc
         dispatcher.register(literal("csignsearch")
             .then(literal("text")
                 .then(argument("query", greedyString())
-                    .executes(ctx -> FindBlockCommand.findBlock(Component.translatable("commands.csignsearch.starting"), predicate(getString(ctx, "query"))))))
+                    .executes(ctx -> FindBlockCommand.findBlock(ctx, Component.translatable("commands.csignsearch.starting"), predicate(getString(ctx, "query"))))))
             .then(literal("regex")
                 .then(argument("query", greedyRegex())
-                    .executes(ctx -> FindBlockCommand.findBlock(Component.translatable("commands.csignsearch.starting"), predicate(getRegex(ctx, "query")))))));
+                    .executes(ctx -> FindBlockCommand.findBlock(ctx, Component.translatable("commands.csignsearch.starting"), predicate(getRegex(ctx, "query")))))));
```

Comment:
We could maybe add a `--keep-searching` option to `/csignsearch` as well while we're at it.

Reply:
oh yeah, never even thought about that